client/js/widgets/widget.js

```diff
@@ -921,6 +921,8 @@ export class Widget extends StateManaged {
         }
         if((a.property == 'parent' || a.property == 'deck') && a.value !== null && !widgets.has(a.value)) {
           problems.push(`Tried setting ${a.property} to ${a.value} which doesn't exist.`);
+        } else if (a.property == "_ancestor") {
```

Comment:
If more pseudo-properties are added, this will become unwieldy. Since users can presumably have custom properties starting with `_`, you can't just check the first character. Perhaps there should be a table of defined custom properties that you can check here.

Reply:
Good point.